conan/tools/cmake/cmakedeps/templates/target_data.py

```diff
@@ -212,7 +212,6 @@ def join_paths_single_var(values):
             return '"%s"' % ";".join(p.replace('\\', '/').replace('$', '\\$') for p in values)
 
         self.include_paths = join_paths(cpp_info.includedirs)
-        self.include_path = join_paths_single_var(cpp_info.includedirs)
```

Comment:
This one is not used in the template (but I didn't check the if it is used in the rest of the codebase)